### ISSUE TITLE
Notebook keybindings for insert cell above and below are not mentioned

### ISSUE THREAD

**Issue by ulugbekna** [microsoft/vscode#213641]
Testing #213343

1. Focus on a notebook cell 
2. Press alt+f1 to bring accesibility help 
3. See keybindings for "insert cell above/below" missing; neither setting a keybinding is mentioned

![Image](https://github.com/microsoft/vscode/assets/16353531/29c2fad5-a691-4e49-9e18-15554ec91f63)

**Comment by meganrogge** [comment]
this is because those don't support keybindings I believe ? cc @amunger

**Comment by amunger** [comment]
I think I was trying to reduce the verbosity by combining those related commands, but multiple commands weren't available in the template

**Comment by meganrogge** [comment]
Not sure what you mean 🤔 ... what template?

**Comment by amunger** [comment]
There used to be a helper function when this was added, https://github.com/microsoft/vscode/pull/186229/files
looks like it's just text now

**Comment by meganrogge** [comment]
oh, I must've missed replacing it with the `<keybinding:commandId>` when I did that refactor

**Comment by karthiknadig** [comment]
![image](https://github.com/user-attachments/assets/fae3182a-50cd-4415-a9a6-00d028fd6d5e)
